src/System.Management.Automation/engine/hostifaces/HostUtilities.cs

```diff
@@ -811,6 +832,192 @@ public static Collection<PSObject> InvokeOnRunspace(PSCommand command, Runspace
+            string actionStyle = PSStyle.Instance.Formatting.FeedbackAction;
+            string ansiReset = PSStyle.Instance.Reset;
+
+            if (!ui.SupportsVirtualTerminal)
+            {
+                captionStyle = string.Empty;
+                italics = string.Empty;
+                nameStyle = string.Empty;
+                textStyle = string.Empty;
+                actionStyle = string.Empty;
+                ansiReset = string.Empty;
+            }
+
+            var output = new StringBuilder();
+            var chkset = new HashSet<FeedbackItem>();
+
+            foreach (FeedbackResult entry in feedbacks)
+            {
+                output.AppendLine();
+                output.Append(captionStyle).Append('[').Append(ansiReset)
+                    .Append(nameStyle).Append(italics).Append(entry.Name).Append(ansiReset)
+                    .Append(captionStyle).Append(']').Append(ansiReset);
+
+                FeedbackItem item = entry.Item;
+                chkset.Add(item);
+
+                do
+                {
+                    RenderText(output, item.Header, textStyle, ansiReset, indent: 2, startOnNewLine: true);
+                    RenderActions(output, item, textStyle, actionStyle, ansiReset);
+                    RenderText(output, item.Footer, textStyle, ansiReset, indent: 2, startOnNewLine: true);
+
+                    // A feedback provider may return multiple feedback items, though that may be rare.
+                    item = item.Next;
+                }
+                while (item is not null && chkset.Add(item));
+
+                ui.Write(output.ToString());
+                output.Clear();
+                chkset.Clear();
+            }
+
+            // Feedback section ends with a new line.
+            ui.WriteLine();
+        }
+
+        /// <summary>
+        /// Helper function to render feedback message.
+        /// </summary>
+        /// <param name="output">The output string builder to write to.</param>
+        /// <param name="text">The text to be rendered.</param>
+        /// <param name="style">The style to be used.</param>
+        /// <param name="ansiReset">The ANSI code to reset.</param>
+        /// <param name="indent">The number of spaces for indentation.</param>
+        /// <param name="startOnNewLine">Indicates whether to start writing from a new line.</param>
+        internal static void RenderText(StringBuilder output, string text, string style, string ansiReset, int indent, bool startOnNewLine)
+        {
+            if (text is null)
+            {
+                return;
+            }
+
+            if (startOnNewLine)
+            {
+                // Start writing the text on the next line.
+                output.AppendLine();
+            }
+
+            // Apply the style.
+            output.Append(style);
+
+            int count = 0;
+            var trimChars = "\r\n".AsSpan();
+            var span = text.AsSpan().Trim(trimChars);
```

Comment:
trimChars is used once and can be const.

Reply:
`ReadonlySpan` cannot be declared as const.